packages/realm-web/src/User.ts

```diff
@@ -121,8 +129,6 @@ export class User<
     }
 
     /**
-     * The automatically-generated internal id of the user.
```

Comment:
TBH, I think the "automatically-generated internal" bit is useful info. I'd add that into the @returns sentence